client/jetpack-cloud/sections/agency-dashboard/sites-overview/sites-dataviews/index.tsx

```diff
@@ -0,0 +1,199 @@
+import { DataViews } from '@wordpress/dataviews';
+import { useTranslate } from 'i18n-calypso';
+import { useCallback } from 'react';
+import SiteActions from 'calypso/jetpack-cloud/sections/agency-dashboard/sites-overview/site-actions';
+import useFormattedSites from 'calypso/jetpack-cloud/sections/agency-dashboard/sites-overview/site-content/hooks/use-formatted-sites';
+import SiteStatusContent from 'calypso/jetpack-cloud/sections/agency-dashboard/sites-overview/site-status-content';
+import TextPlaceholder from 'calypso/jetpack-cloud/sections/partner-portal/text-placeholder';
+import { AllowedTypes, Site, SiteData } from '../types';
+
+import './style.scss';
+
+interface Props {
+	data:
+		| { sites: Array< Site >; total: number; perPage: number; totalFavorites: number }
+		| undefined;
+	isLoading: boolean;
+}
+
+const SitesDataViews = ( { data, isLoading }: Props ) => {
+	const translate = useTranslate();
+
+	const view = {
+		type: 'table',
+		perPage: 10,
+		page: 1,
+		sort: {
+			field: 'site',
+			direction: 'desc',
+		},
+		search: '',
+		filters: [ { field: 'status', operator: 'in', value: 'Needs attention' } ],
+		hiddenFields: [ 'status' ],
+		layout: {},
+	};
+
+	const sites = useFormattedSites( data?.sites ?? [] );
+
+	const renderField = useCallback(
+		( column: AllowedTypes, item: SiteData ) => {
+			if ( isLoading ) {
+				return <TextPlaceholder />;
+			}
+
+			if ( column ) {
+				return (
+					<SiteStatusContent
+						rows={ item }
+						type={ column }
+						isLargeScreen
+						isFavorite={ item.isFavorite }
+						siteError={ item.site.error }
+					/>
+				);
+			}
+		},
+		[ sites ]
```

Comment:
We are getting a warning in this line about a missing dependency (`isLoading`).

Reply:
I'll add the dependency.

Reply:
https://github.com/Automattic/wp-calypso/pull/87250/commits/bd5a383a45581553f04532907ec8d0eda49d252e